pages/common/simplehttpserver.md

```diff
@@ -0,0 +1,25 @@
+# simplehttpserver
+
+> A simple HTTP/S server that support file upload, basic auth, and YAML rules for custom responses.
+> A Go alternative to Python's http.server.
```

Comment:
```suggestion
> A Go alternative to Python's http.server.
> Warning: this project is intended for development purposes only, it may contain security issues.
```